### PR TITLE
Update JetPack SDK

### PR DESCRIPTION
**Pull request by DanielHo-BS**

### CODE REVIEW

File: nvidia/packages/jetpack_sdk/jetpack_sdk_install.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    sudo apt update
+    sudo apt install nvidia-l4t-jetson-multimedia-api
+
+.. warning:: 
+    | It's **not suggested** to install ``nvidia-jetpack`` directly, because nvidia-jetpack is a meta package which will install **all JetPack SDK** compoments which occupy **8GB**.
+    | You can only install ``particular`` JetPack SDK compoment.
+
+2. Install the whole JetPack SDK
+--------------------------------
+Assuming your Jetson developer kit has been flashed with and is running **L4T 32.3.1** or higher,the following commands wil install **all** ``JetPack``.
```
Comment (evshary) [inline nvidia/packages/jetpack_sdk/jetpack_sdk_install.rst]: wil -> will

File: nvidia/packages/jetpack_sdk/jetpack_sdk_yolo.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+5. Download the pre-trained weights.
+------------------------------------
+
+.. code-block:: bash
+
+    wget https://pjreddie.com/media/files/yolov3-tiny.weights
+
+.. note::
+
+    ADLINK doesn't own the pre-train data. Thsi pre-trained data is a contribution from original author in community.
```
Comment (evshary) [inline nvidia/packages/jetpack_sdk/jetpack_sdk_yolo.rst]: pre-train -> pre-trained

Comment (evshary) [reply nvidia/packages/jetpack_sdk/jetpack_sdk_yolo.rst]: Thsi -> This

File: nvidia/packages/jetpack_sdk/index.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.. _jetpack_sdk:
+
+The Jetpack SDK
```
Comment (evshary) [inline nvidia/packages/jetpack_sdk/index.rst]: Remove The here.

**Review by evshary — CHANGES_REQUESTED** [review]

File: nvidia/packages/jetpack_sdk/jetpack_sdk_yolo.rst (hunk 2 above)
Comment (DanielHo-BS) [reply nvidia/packages/jetpack_sdk/jetpack_sdk_yolo.rst]: Done

File: nvidia/packages/jetpack_sdk/jetpack_sdk_install.rst (hunk 1 above)
Comment (DanielHo-BS) [reply nvidia/packages/jetpack_sdk/jetpack_sdk_install.rst]: Done

File: nvidia/packages/jetpack_sdk/index.rst (hunk 3 above)
Comment (DanielHo-BS) [reply nvidia/packages/jetpack_sdk/index.rst]: Done

**Review by DanielHo-BS — COMMENTED** [review]
Finished these request changes.

**Pull request merged**